docs/layouts/shortcodes/table.html

```diff
@@ -0,0 +1,322 @@
+{{/*
+https://github.com/pablotron/hugo-shortcode-table
+
+Custom Hugo table shortcode which renders tables by:
+
+1. Reading table definition and data from the front matter or
+site data directory.
+2. Renders the table using CSS classes rather than inline styles.
+
+Why does this shortcode exist?
+
+1. The default Markdown table markup in Hugo is horrible for anything
+more than rudimentary tables.
+2. The default Markdown table renderer in Hugo does not give you a way
+to set cell alignment without using inline styles, which means
+enabling `unsafe-inline` in your `Content-Security-Policy` header.
+3. I prever specifying table contents in an external structured file
```

Comment:
```suggestion
3. I prefer specifying table contents in an external structured file
```
Maybe also remove references to "I". Who this is referring to will become unclear over time.